### ISSUE TITLE
The compound loss may have incorrect magnitudes

### ISSUE THREAD

**Issue by rzuberi** [rzuberi/distillCellSegTrack#7]
Our KD_loss in `train_utils.py` calculates the model's output loss which consists of the flows in X, the flows in Y and the probability map by finding the MSE loss for the flows and a BCE with logits loss for the probability map and summing both losses.

It has been suggested that these two losses may be of different magnitudes, making the model concentrate on improving the probability map and not the flows.

We need to train a model and plot these losses individually to inspect if they are of different magnitudes. The proposed solution would then be to add weights to each loss.

**Comment by rzuberi** [comment]
I found that Cellpose was first multiplying the ground truth of the flows by 5 and then getting the MSE loss, I have now implemented that. This made the flow loss even larger, around 80 in the first epochs, whereas the probability map loss starts at around 1 and goes down to 0.05 after a few epochs. Therefore, I divide the flow los by 80 before summing them.
